schemas/core/documents/dialog.ts

```diff
@@ -123,10 +123,10 @@ export default function dialog({
         of: items,
       },
       {
-        title: "Page Headers",
+        title: "Dialog Headers",
         name: "headers",
         type: "array",
-        of: headers,
```

Comment:
We have headers, getting from the root level. Please use the same. Provide _dialogHeader_ as header. Check **FeatureSchemaProperties** for more details regarding how to provide  _dialogHeader_ to header.